migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt

```diff
@@ -44,9 +44,9 @@ class MigrationManagerIntegrationTest {
     // exceed their declared trust limit for the asset being sent.
     private val LINE_FULL_RESULT_CODE = "op_line_full"
 
-    private val timeoutDurationSecondsLong: Long = 45 //TODO change to 15
-    private val timeoutDurationSecondsShort: Long = 30 //TODO change to 10
-    private val timeoutDurationSecondsVeryShort: Long = 20 //TODO change to 2
+    private val timeoutDurationSecondsLong: Long = 450 //TODO change to 15
```

Comment:
45 changed to 450 and we have a comment "change to 15", I don't know who to believe to...

Reply:
ha ha ha
anyway we have a branch for handling tests but i will change it here also.